scout/run.py

```diff
@@ -3699,23 +3920,33 @@ def compete_adj(
                 adj["carbon"]["total"][x][yr] = [
                 (x[yr] * adj_t[v]) for x, v in zip(
                     adjlist[3:5], ["energy", "carbon"])]
+            # Adjust efficient-captured energy if these data are present
+            if x == "efficient":
+                try:
+                    mast["energy"]["total"]["efficient-captured"][yr] = \
+                        mastlist[10][yr] - (
+                        adjlist[10][yr] * (1 - adj_t["energy-captured"]))
+                    adj["energy"]["total"]["efficient-captured"][yr] = (
+                        adjlist[10][yr] * adj_t["energy-captured"])
+                except (KeyError, IndexError):
+                    pass
 
             # Adjust total emissions from fugitive emissions if applicable
             if measure.fug_e:
                 # Adjust fugitive methane emissions results if applicable
                 if measure.fug_e and "methane" in measure.fug_e:
                     # Total
                     mast["fugitive emissions"]["methane"][
-                        "total"][x][yr] = mastlist[10][yr] - (
-                            adjlist[10][yr] * (1 - adj_t["energy"]))
+                        "total"][x][yr] = mastlist[-4][yr] - (
```

Comment:
This approach is not new to this PR, but seems risky to count on the correct order of `mastlist` and access values by specific indices, any change to mastlist would require updates to each reference

Reply:
Yes, that's correct, it does require a change which could introduce problems. Any suggestions for how to do it otherwise?

Reply:
Probably make it a dictionary, looking at the code that might get a little too involved to include right now.